<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PendingListWidget</class>
 <widget class="QWidget" name="PendingListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>347</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>347</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PendingListTitleWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 1px solid #ededed;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PendingListView" name="listView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>268</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>348</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
	background-color: rgb(250, 250, 250);
	border-top: 1px solid #ededed;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="upPageButton">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>37</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:none;
	background-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
	background-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
background-image: url(:/saleImage/btn-up-disable.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ededed;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downPageButton">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>37</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:none;
	border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled
{
	border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pageNumLabel">
        <property name="text">
         <string>1/1é¡µ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PendingListTitleWidget</class>
   <extends>QWidget</extends>
   <header>PendingListTitleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PendingListView</class>
   <extends>QListView</extends>
   <header>PendingListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
